src/Markdig/Helpers/LinkHelper.cs

```diff
@@ -1006,7 +1006,7 @@ private static bool IsEndOfUri(char c, bool isAutoLink)
         return c == '\0' || c.IsSpaceOrTab() || c.IsControl() || (isAutoLink && c == '<'); // TODO: specs unclear. space is strict or relaxed? (includes tabs?)
     }
 
-    public static bool IsValidDomain(string link, int prefixLength)
+    public static bool IsValidDomain(string link, int prefixLength, bool allowDomainWithoutPeriod)
```

Comment:
```suggestion
    public static bool IsValidDomain(string link, int prefixLength, bool allowDomainWithoutPeriod = false)
```

To keep it source compatible since these are public helpers